Recent gcc is more strict against undeclared function calls
like system() and exit(), breaking gtk version check.
Also fix signed-unsigned-comparison warning.
--- aclocal.m4.orig	2011-05-25 16:55:15.609086191 +0200
+++ aclocal.m4	2011-05-25 16:55:22.999136294 +0200
@@ -765,6 +765,7 @@
       AC_TRY_RUN([
 #include <gtk/gtk.h>
 #include <stdio.h>
+#include <stdlib.h>
 
 int 
 main ()
 {
-  int major, minor, micro;
+  guint major, minor, micro;
 
   system ("touch conf.gtktest");
 
